src/coreModules/api/apiClient/preProcess/index.js

```diff
@@ -20,9 +22,25 @@ export default function preProcess({
     requestData,
   })
 
+  const pathParams = createPathParams({
+    apiConfig,
+    endpointConfig,
+    methodConfig,
```

Comment:
is it unnecessary to pass `apiConfig` and `methodConfig` since they're not used in those two methods?

Reply:
Usually yes but it this case i think its ok. Will motivate face to face :)

Reply:
well at least I can see the consistency and it doesn't hurt :-)